### ISSUE TITLE
Draft plan for praqma.com work in Q1

### ISSUE THREAD

**Issue by monaslarsen** [Praqma/praqma.com#519]
Mona will work much more on praqma.com from now on.
Let's align what content to prioritize

**Comment by meekrosoft** [comment]
Great!  Is the idea to use this for brainstorming?  Should I add my 2kr here?

**Comment by monaslarsen** [comment]
Yes @meekrosoft, add away!

**Comment by jonnycoull** [comment]
@monaslarsen @meekrosoft ill add my ROAD MAP here once its finished being written. that will hep form a plan, also mona do you have a list of planned events and dates or would that be @JasmineCrozier now? cheers.

**Comment by monaslarsen** [comment]
That would be Miss Event Manager Jasmine, @jonnycoull :)

**Comment by JasmineCrozier** [comment]
@jonnycoull Yes we are working on a 2017 event plan, but dates are not set yet.
To give you an overview, there should be around:
- 3 CoDe Conf (one in each capital city in Q3)
- 6 CoDe Academy (Q2-Q3)
- 2 Atlassian Accelerator Day (Q1)
- 2 DevOps Days (Q2)
- 3 or 4 Day of ... 
- 2 CoDe Alliance gathering
- Many many meet-ups (if I manage to gather all information, we hope for 5 or more per city)

**Comment by meekrosoft** [comment]
Just thinking out loud here: maybe we don't want our marketing strategy discussed in public?  I will send my input via mail...
